src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,11 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+
+        for (int i = 0; i < 3; i++) {
+            Ball ball = lottery.getRandomBall();
+            System.out.println(ball.toString());
```

Comment:
1. method .toString() is called automatically for println() command, no need to explicitly set it:
```suggestion
            Ball ball = lottery.getRandomBall();
            System.out.println(ball);
```
2. There is no real benefit of introducing new variable to use it only once and right here:
```suggestion
            System.out.println(lottery.getRandomBall());
```